plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/ProductVariationTemplate.php

```diff
@@ -0,0 +1,574 @@
+<?php
+/**
+ * SimpleProductTemplate
+ */
+
+namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+
+/**
+ * Simple Product Template.
+ */
+class ProductVariationTemplate extends AbstractProductFormTemplate implements ProductFormTemplateInterface {
```

Comment:
I'd like to see us explore using conditional visibility when it is available to use the same template for both simple products and variations. Because, they seem very similar.

If not, we should carefully consider when we change ids for things like groups, sections, and fields that appear "the same" in both. Because, if they have the same ids in multiple templates, it will make extensibility easier (for the case where you want to do the same thing in all templates).

We should also consider using composition to build up both templates from common "building blocks" where they have similarity. This will at least allow us to not duplicate too much.

Reply:
> I'd like to see us explore using conditional visibility when it is available 

@mattsherman, I assume that [the relevant discussion](https://github.com/woocommerce/woocommerce/discussions/40399) has concluded FOR using conditional visibility to manage different product types. Is that right? Are you planning to wrap up/summarize the discussion or plan to wait for more input.

@louwie17 @mdperez86 
Should we move forward with implementing templates (since it is useful on its own) and migrating it to the alternative approach later, if decided? How much effort would it be to change it?
Given the ETA of conditional visibility, my concern is that it could become the blocker to shipping this work. 
Thoughts?

Reply:
Yup, that would be cool.

Is it currently possible to create multiple `FormTemplates` for this, like a specific `FormTemplate` for a section? and use that across templates?

Reply:
>Should we move forward with implementing templates (since it is useful on its own) and migrating it to the alternative approach later, if decided? How much effort would it be to change it?
Given the ETA of conditional visibility, my concern is that it could become the blocker to shipping this work.
Thoughts?

Yes, we are going to continue without the conditional block here. We don't have to much time to release this and decisions like that are going to be covered in further project and before the issues are being placed to InProgress. On the other hand we should identify first/better what is the ideal scenario to use conditional or template before starting to use it.

Reply:
> Is it currently possible to create multiple FormTemplates for this, like a specific FormTemplate for a section? and use that across templates?

There are a lot of thins we can do here to avoid duplications, I do agree with @mattsherman that a composition is a strong candidate here, but reusing or abstracting groups of blocks is going to require the design to be aligned across product types. This is of course and improvement topic we can discuss in more details in the future.

Reply:
> I assume that https://github.com/woocommerce/woocommerce/discussions/40399 has concluded FOR using conditional visibility to manage different product types. Is that right? Are you planning to wrap up/summarize the discussion or plan to wait for more input.

We should include guidance around this in the UX/dev guidelines. I'll make a note to follow up.

> Is it currently possible to create multiple FormTemplates for this, like a specific FormTemplate for a section? and use that across templates?

Not currently (I have thought about 'template fragments'). Currently, the best that can be done is to use composition with PHP when building the template. Which isn't bad... I just think it could be simplified a bit via something like fragments.

> Yes, we are going to continue without the conditional block here. 

I think that is a reasonable decision in the short term. I don't think the effort to move to a conditional template would be that great if the two templates really are very similar (which is the scenario where a conditional template would make sense).